services/tmc/public/runWorkerTemplate.py

```
@@ -0,0 +1,72 @@
+    return await inputPromise(prompt)
+
+
+globals()["input"] = input_impl
+
+
+async def wrap_execution():
+    try:
+        await execute()
+        exit()
+    except Exception:
+        t, v, tb = sys.exc_info()
```

Ruff (0.15.2)

[warning] Unpacked variable `t` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)